x/cron/keeper/msg_server.go

```diff
@@ -0,0 +1,133 @@
+package keeper
+
+import (
+	"context"
+	errorsmod "cosmossdk.io/errors"
+	"fmt"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/dymensionxyz/rollapp-wasm/x/cron/types"
+)
+
+type msgServer struct {
+	Keeper
+}
+
+// NewMsgServerImpl returns an implementation of the MsgServer interface
+// for the provided Keeper.
+func NewMsgServerImpl(keeper Keeper) types.MsgServer {
+	return &msgServer{Keeper: keeper}
+}
+
+var _ types.MsgServer = msgServer{}
+
+func (k msgServer) RegisterCron(goCtx context.Context, msg *types.MsgRegisterCron) (*types.MsgRegisterCronResponse, error) {
```

Comment:
can you please make `RegisterCron`, `UpdateCronJob` and `DeleteCronJob` go through governance proposals instead of being permissioned, as suggested previously